lib/cog/commands/alias/helpers.ex

```diff
@@ -101,7 +101,13 @@ defmodule Cog.Commands.Alias.Helpers do
     do: EctoJson.render(data)
 
   # Special errors that come when there are issues with the database.
-  defp db_errors(errors),
-    do: Enum.map_join(errors, "\n", fn({key, message}) -> "#{key} #{message}" end)
+  defp db_errors(errors) do
+    Enum.map_join(errors, "\n", fn
+      ({:name, "has already been taken"}) ->
```

Comment:
Might be better to fix this in the model, instead: [unique_constraint/3](https://hexdocs.pm/ecto/2.0.0-beta.1/Ecto.Changeset.html#unique_constraint/3) (see the `:message` option)